### PR TITLE
Remove javascript runtime

### PR DESCRIPTION
**Pull request by CarstenWalther**
Removes the javascript runtime.
Also remove two libraries uglifier and coffee-rails that required the runtime.
Coffee-rails is for .coffee files that are not used.

Uglifier was used for javascript compression, which is not that import in this stage of the project.

Main reason for this change is to reduce docker image build times.
This PR reduces it from 2:15 min to 1:12 min.

### CODE REVIEW

File: config/environments/production.rb [hunk 1]
```diff
@@ -25,7 +25,7 @@
   config.serve_static_files = ENV['RAILS_SERVE_STATIC_FILES'].present?
 
   # Compress JavaScripts and CSS.
-  config.assets.js_compressor = :uglifier
+  # config.assets.js_compressor = :uglifier
```
Comment (Bouncner) [inline config/environments/production.rb]: Aren't there any none-js compressors that do not screw the build time?

Comment (CarstenWalther) [reply config/environments/production.rb]: I couldn't find an alternative javascript compressor to uglifier. It is used by default when using ruby on rails.

Javascript files are only sent when a user navigates to the consumer documentation. And in that case, most of the javascript is already minified.
The consumer's core tasks (buying products and providing an API) don't use javascript.
That's why I think it's ok to not use a javascript compressor.

Comment (Bouncner) [reply config/environments/production.rb]: Fair enough.

Comment (Bouncner) [reply config/environments/production.rb]: Nice catch with that JS runtime.

**Pull request merged**